maquetado/scss/footer.scss

```diff
@@ -0,0 +1,21 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 14px;
+  width: 100%;
+}
+
+.footer-title {
+  color: $white;
+  font-size: $text-big;
+  margin-bottom: 14px;
+}
+
+.footer-text {
+  align-self: flex-end;
+  color: $white;
+  font-size: $text-small;
+  margin-top: 14px;
```

Comment:
It's preferred to add this as a margin bottom in the footer-wolox-icon, to be consistent with the rest and also to make it easier to read. When you check the html in the inspector you can easily see that the image is separated from the text by 14px at the bottom.